versioned_docs/version-0.9.0/agents/test-agent.mdx

```diff
@@ -5,6 +5,163 @@ title: Test Agent
 
 # Test Agent
 
-Drill4J Test Agent for Java enables integration with testing frameworks.
+The Drill4J Test Agent allows you to map tests to code coverage and vice versa.
+Key Features:
+- Test context propagation
+- Control over test execution
 
-> 🚧 UNDER CONSTRUCTION. TO BE COMPLETED SOON 🚧
+General integration steps are:
+
+1. [Download Drill4J agent files](#how-to-download)
+2. [Set path to agent when running tests](#how-to-set-path-to-agent)
+3. [Set agent parameters](#how-to-set-agent-parameters)
+
+Due to the diverse tools used in software development — such as CI/CD providers, deployment methods, testing stages — the exact process will depend on the environment.
+
+## How to download
+
+### Download agent files manually
+
+The most basic way to get agent files is to download release .zip manually.
+
+1. Download release appropriate for your platform at  <https://github.com/Drill4J/autotest-agent/releases/tag/v0.23.1>.
+
+    > `mingwX64` is for Windows, Linux and Mac releases are called respectively.
+
+2. Unzip the file. Rename unzipped folder to `agent`.
+
+3. Inside you'll find the following files:
+
+    -  `libdrill_agent.so` (on Windows - `.dll`, on Mac - `.dylib`)
+    - `drill-runtime.jar`
+    - `drill.properties`
+
+Refer to [next section](/docs/agents/autotest-agent#how-to-set-path-to-agent) for further steps.
+
+### Use Drill4J CI/CD Integration Plugin for Maven and Gradle
```

Comment:
I suggest to switch this with `Download agent files manually` and highlight that this is the primary way to use Test Agent